bin/dartdoc.dart

```diff
@@ -98,6 +98,9 @@ Future<void> main(List<String> arguments) async {
     }, onError: (e, Chain chain) {
       if (e is DartdocFailure) {
         stderr.writeln('\ndartdoc failed: ${e}.');
+        if (config.verboseWarnings) {
+          stderr.writeln(chain.terse);
```

Comment:
Hmmm.  I suppose we can use verboseWarnings for this, though it wasn't really what it was for originally.